src/Microsoft.Dnx.Project/project.json

```diff
@@ -36,5 +36,14 @@
                 "System.Threading.Tasks": "4.0.11-beta-*"
             }
         }
+    },
+    "scripts": {
+        "postbuild": [
```

Comment:
It looks like it was just missed, mostly because no one works with this project :)